src/main/java/ru/teti/springcourse/controllers/PeopleController.java

```diff
@@ -15,20 +14,19 @@
 public class PeopleController {
     private final PersonDAO personDAO;
 
-    @Autowired
     public PeopleController(PersonDAO personDAO) {
         this.personDAO = personDAO;
     }
 
     @GetMapping()
     public String index(Model model) {
-        model.addAttribute("people", personDAO.index());
+        model.addAttribute("people", personDAO.getAll());
```

Comment:
I believe, getAll() is much more meaningful name for such method at the DAO level